python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -285,8 +289,14 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
                   )
               )}
             </React.Fragment>
+          ) : actor.invalidStateType === 'infeasibleActor' ? (
+            <span className={classes.invalidStateTypeInfeasible}>
+              {actor.actorTitle} is an infeasible actor.
```

Comment:
Let's change this to `{actor.actorTitle} is infeasible.` to be consistent with the message for pending actors.

Reply:
Changed! I also added a description regarding `infeasible actor`.
```
 (Infeasible actor means an actor cannot be created because 
              Ray cluster cannot satisfy resources requirement).
```